example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,39 +1,47 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="io.maido.intercomexample">
 
-    <!-- The INTERNET permission is required for development. Specifically,
-         flutter needs it to communicate with the running application
-         to allow setting breakpoints, to provide hot reload, etc.
-    -->
-    <uses-permission android:name="android.permission.INTERNET"/>
+  <!-- The INTERNET permission is required for development. Specifically,
+       flutter needs it to communicate with the running application
+       to allow setting breakpoints, to provide hot reload, etc.
+  -->
+  <uses-permission android:name="android.permission.INTERNET" />
 
-    <!-- io.flutter.app.FlutterApplication is an android.app.Application that
-         calls FlutterMain.startInitialization(this); in its onCreate method.
-         In most cases you can leave this as-is, but you if you want to provide
-         additional functionality it is fine to subclass or reimplement
-         FlutterApplication and put your custom class here. -->
-    <application
-        android:name="io.flutter.app.FlutterApplication"
-        android:label="intercom_example"
-        android:icon="@mipmap/ic_launcher">
-        <activity
-            android:name=".MainActivity"
-            android:launchMode="singleTop"
-            android:theme="@style/LaunchTheme"
-            android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection|fontScale|screenLayout|density"
-            android:hardwareAccelerated="true"
-            android:windowSoftInputMode="adjustResize">
-            <!-- This keeps the window background of the activity showing
-                 until Flutter renders its first frame. It can be removed if
-                 there is no splash screen (such as the default splash screen
-                 defined in @style/LaunchTheme). -->
-            <meta-data
-                android:name="io.flutter.app.android.SplashScreenUntilFirstFrame"
-                android:value="true" />
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN"/>
-                <category android:name="android.intent.category.LAUNCHER"/>
-            </intent-filter>
-        </activity>
-    </application>
+  <!-- io.flutter.app.FlutterApplication is an android.app.Application that
+       calls FlutterMain.startInitialization(this); in its onCreate method.
+       In most cases you can leave this as-is, but you if you want to provide
+       additional functionality it is fine to subclass or reimplement
+       FlutterApplication and put your custom class here. -->
+  <application
+      android:icon="@mipmap/ic_launcher"
+      android:label="intercom_example">
+    <activity
+        android:name=".MainActivity"
```

Comment:
As per the [migration guide](https://github.com/flutter/flutter/wiki/Upgrading-pre-1.12-Android-projects), since `MainActivity.kt` is empty, you can remove that file and change the activity name reference here.

```suggestion
        android:name="io.flutter.embedding.android.FlutterActivity"
```